blablacar-persistence/src/main/java/cz/fi/muni/pa165/teamred/entity/Comment.java

```diff
@@ -33,13 +32,15 @@
     @Column(nullable=false)
     private String text;
 
-//    @ManyToOne(fetch=FetchType.LAZY)
-//    @JoinColumn(name="USER_ID")
-//    private User author;
-//
-//    @ManyToOne(fetch=FetchType.LAZY)
-//    @JoinColumn(name="RIDE_ID")
-//    private Ride ride;        
+    @NotNull
```

Comment:
Toto som odkomentoval ja v mojom pull requeste s fixnutymi anotaciami. NotNull som nedaval, lebo comment by mhol byt v pripade author == null oznaceny ako anonym a ride == null je na zvazenie, moze byt a nemusi. Tieto zmeny by si ale mohol deletnut z pull requestu nech sa to nepobije.

Reply:
Tu som neriesil spravnost @ManyToMany ... ale aby sa ten projekt skompiloval. Napriklad mi nezozere entitu User bez zmeny mena tabulky.

Reply:
aha takto.. no ja som to vyriesil tak ze som si tiez pomenil ine classy kvoli skompilovatelnosti, ale commitol a dal do PR len tie ktore som mal implementovat/chcel zmenit aj v deve. A tie ostatne si revertnem a nataham od ostatnych ked to budu mat